tests/python/gaia-ui-tests/gaiatest/tests/unit/test_wifi_multiple_connect.py

```diff
@@ -0,0 +1,17 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from marionette.marionette_test import parameterized
+
+from gaiatest import GaiaTestCase
+
+
+class TestWiFiMultipleConnect(GaiaTestCase):
+
+    @parameterized("1st_run")
+    @parameterized("2nd_run")
```

Comment:
I'd add a comment to explain why we need a second run of this test. Without this information we might end up changing the test to:

``` python
    def test_connect_and_forget_all_networks(self):
        for x in range(0, 60):
             # blah blah
```